ledger/accountdb.go

```diff
@@ -631,6 +631,46 @@ func accountsAddNormalizedBalance(tx *sql.Tx, proto config.ConsensusParams) erro
 	return rows.Err()
 }
 
+// removeEmptyAccountData removes empty AccountData msgp-encoded entries from accountbase table
+// and optionally returns list of addresses that were eliminated
+func removeEmptyAccountData(tx *sql.Tx, queryAddresses bool) (num int64, addresses []basics.Address, err error) {
+	if queryAddresses {
+		rows, err := tx.Query("SELECT address, data FROM accountbase where length(data) = 1 and data = x'80'") // empty AccountData is 0x80
```

Comment:
you're not using the data argument. maybe eliminate it ?

Reply:
fixed